src/test/java/nextstep/subway/auth/acceptance/AuthAcceptanceTest.java

```diff
@@ -1,24 +1,90 @@
 package nextstep.subway.auth.acceptance;
 
+import com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import nextstep.subway.AcceptanceTest;
+import nextstep.subway.auth.dto.TokenRequest;
+import nextstep.subway.auth.dto.TokenResponse;
+import nextstep.subway.member.MemberAcceptanceTest;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import static nextstep.subway.member.MemberAcceptanceTest.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class AuthAcceptanceTest extends AcceptanceTest {
+    @Override
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+        ExtractableResponse<Response> createResponse = MemberAcceptanceTest.회원_생성을_요청(EMAIL, PASSWORD, AGE);
```

Comment:
사용하지 않는 변수인 것 같은데 값을 뽑아내지 않아도 될 것 같습니다 !
```suggestion
        MemberAcceptanceTest.회원_생성을_요청(EMAIL, PASSWORD, AGE);
```